libs/util-prohibited-data/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import { toLower } from 'lodash';
+
+import prohibitedAddresses from './lib/prohibited-addresses.json';
+
+type Address = `0x${string}`;
+
+export const isAddressProhibited = (address: Address | undefined) => {
```

Comment:
does it make sense to move this function right to the `libs/util-prohibited-data`?